### ISSUE TITLE
I am getting a message like this: "An error occurred. Please try again later.(Playback ID: W.....cC)"

### ISSUE THREAD

**Issue by priyankagupta23pg** [PierfrancescoSoffritti/android-youtube-player#549]
I am passing video id but still getting this error on playing video using this library. while using that video id video is playing in official youtube player. So need to remove this library from my project

**Comment by m-akopov** [comment]
The same happens with me :/

**Comment by m-akopov** [comment]
I noticed it happens with some of videos only. 90% of videos are pretty good

**Comment by PierfrancescoSoffritti** [comment]
@m-akopov can you please confirm that it happens only when playing programmatically? It seems very strange to me.

My understanding of this issue is that YouTube does not allow certain videos to be played on an embedded player.

**Comment by m-akopov** [comment]
@PierfrancescoSoffritti  

Yes I confirm. From XML is fine. I tried everything, even set ReadyListener after 2000 milliseconds :D and then onReady cueVideo, or loadVideo. But without success.

Now I have <WebView> and embedded video in iFrame with HTML and the same problematic videos(and all others) work fine.

**Comment by PierfrancescoSoffritti** [comment]
Can you send me a few ids of videos that have this problem?

**Comment by m-akopov** [comment]
Sure, here are videos:

https://youtu.be/Dzbb0OWnl2A
https://youtu.be/X1Jqv1bESS8

**Comment by aalloul** [comment]
I was having the same issue as @m-akopov but I think I found a way to solve it. Here's how I'm setting the videoID programmatically:
```
    private fun setupVideo(vidId: String) {
        val youtubePlayerView: YouTubePlayerView = binding.resourceVideo
        lifecycle.addObserver(youtubePlayerView)
        youtubePlayerView.enterFullScreen()
        youtubePlayerView.addYouTubePlayerListener(object : AbstractYouTubePlayerListener() {
            override fun onReady(youTubePlayer: YouTubePlayer) {
                youTubePlayer.loadVideo(vidId, 0f)
                youTubePlayer.play()
            }
        })
    }
```
On the XML side I removed the `autoPlay` and `videoId` properties.

**Comment by vooda--** [comment]
In my case, this error was when I tried to use youtube links instead of video Id.

**Comment by arti-pandey** [comment]
i solve this issue just removing autoplay and viodeo id from xml file
